build_tools/benchmarks/shark/run_shark.sh

```diff
@@ -84,7 +96,40 @@ rm -rf ~/.local/shark_tank
 PYTHON=python3.10 VENV_DIR=iree.venv BENCHMARK=1 IMPORTER=1 USE_IREE=1 ./setup_venv.sh
 source iree.venv/bin/activate
 
-export IREE_VERSION=`pip show iree-compiler | grep Version | sed -e "s/^Version: \(.*\)$/\1/g"`
+export IREE_VERSION=$(pip show iree-compiler | grep Version | sed -e "s/^Version: \(.*\)$/\1/g")
+
+if [[ "${USE_LOCAL_IREE}" == true ]]; then
```

Comment:
Wouldn't `USE_LOCAL_IREE` mean using the IREE version already installed? It seems like this is the opposite somehow. Also, is the `IREE_SOURCE_DIR` only used in this case? I think it would be reasonable to key this on that variable being set.

Reply:
`USE_LOCAL_IREE` means use a locally built version of IREE instead of an IREE nightly release. `IREE_SOURCE_DIR` is only used when `USE_LOCAL_IREE` is set to true. The reason why I didn't key on `IREE_SOURCE_DIR` is that this can be blank if the current directory is the source dir. I tried to explicitly set the source dir from the workflow yml by using `GITHUB_WORKSPACE` as the value to `IREE_SOURCE_DIR` but when cd'ing into it from the docker image, it couldn't find it.

Reply:
It seems weird to install and then uninstall this here, but I guess shark doesn't have an option to set up its dependencies without this. Ah well.

If you wanted to explicitly specify the current dir, wouldn't you set `IREE_SOURCE_DIR` to "."? Regarding docker, the `docker_run` script mounts the workspace under a consistent path /work, not the path it happent to be on the host machine. This makes some things work better. In particular, CMake build directories, which contain absolute paths, getting used across machines.

Reply:
SHARK does have an option to not install the `iree-compiler` and `iree-runtime` but it still ends up installing it due to other dependencies/config files.

Oh yes, let me try using "." for `IREE_SOURCE_DIR`.

Reply:
Lol whoops :laughing: That seems like a bug.